extensions/amp-bodymovin-animation/validator-amp-bodymovin-animation.protoascii

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-bodymovin-animation
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-bodymovin-animation"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-bodymovin-animation>
+  html_format: AMP
+  tag_name: "AMP-BODYMOVIN-ANIMATION"
+  requires_extension: "amp-bodymovin-animation"
+
+  attrs: {
+    name: "loop"
+    value_regex: "false|number|true"
```

Comment:
change `value_regex` to `value_regex_casei` and update regex to `(false|number|true)`